.travis.yml

```diff
@@ -19,10 +19,14 @@ sudo: false
 language: python
 
 python:
-  - "2.7"
-  - "3.4"
   - "3.5"
```

Comment:
moderate: remove 3.5

Reply:
Rather than blindly removing Python 3.5 in all the packages I went through some reasoning based on the following facts:
1. The [RFC](https://github.com/inveniosoftware/rfcs/blob/master/framework/0016-python2.md) doesn't say anything about removing Python 3.5.
1. It was my understanding that we are removing Python 3.5 because of Flask (see https://github.com/inveniosoftware/flask-breadcrumbs/issues/48#issue-611341675) and this package doesn't depend on Flask.
3. This package seemed to me to be independent (previous point) so having wider compatibility would be great as it could run, for example, in scripts (this is more a gut feeling than a fact as I am not familiar with the module).

Reply:
From my perspective it is implicitly said in the RFC. In the travis example, it removes the version 3.5. It is true that it is not explicit.

IMO is more a matter of consistency and pragmatism:
- Pragmatism: Python 3.5 has a few months of life ahead and then is gone. It also incurs more Travis cycles.
- Consistency: It has been removed from all flask related repos -as you mention because of Flask itself-.

Since Invenio per-se is stop testing for Python 3.5. The options are:
- Removal
- Allowed failures

I would say that if something is broken we will reactively fix it, but we will not block releases for that reason. Therefore, I would opt for the first - removing it -.

Reply:
Amending, thanks!